spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/Engine.scala

```diff
@@ -102,15 +104,8 @@ object Engine {
   }
 
   private val logger = Logger.getLogger(getClass)
-  private val singletonCounter: AtomicInteger = new AtomicInteger(0)
-  private[bigdl] var localMode: Boolean = {
-    val env = System.getenv("BIGDL_LOCAL_MODE")
-    if(env == null) {
-      false
-    } else {
-      true
-    }
-  }
+  private val singletonCounter = new AtomicBoolean()
```

Comment:
Some of this was a bit of simplification I indulged in while also touching up the docs and getting to know this code. Is it reasonable?